.github/workflows/cibuildwheel-before-build.sh

```diff
@@ -19,11 +19,23 @@ TBB_ZIP="v${TBB_VERSION}.zip"
 curl -L -O "https://github.com/oneapi-src/oneTBB/archive/refs/tags/${TBB_ZIP}"
 unzip -q "${TBB_ZIP}"
 
+#
+EXTRA_CMAKE_ARGS=""
+if [ "${PLATFORM}" = "macos" ]; then
+    if [[ ${ARCHFLAGS} == *"arm64"* ]]; then
+        EXTRA_CMAKE_ARGS="-DCMAKE_OSX_ARCHITECTURES=arm64"
+    fi
+fi
+echo "EXTRA_CMAKE_ARGS=${EXTRA_CMAKE_ARGS}"
+
+# clean the build to rebuild for arm64
+rm -rf "${PACKAGE_DIR}/tbb"
+
 # Move to a hard-coded path (defined by CIBW_ENVIRONMENT)
 mv "oneTBB-${TBB_VERSION}" "${PACKAGE_DIR}/tbb"
 cd "${PACKAGE_DIR}/tbb"
 mkdir -p build
 cd build
-cmake ../ -DTBB_TEST=OFF -DTBB_STRICT=OFF
-cmake --build . -j
+cmake ../ -DTBB_TEST=OFF -DTBB_STRICT=OFF -DCMAKE_BUILD_TYPE=Release ${EXTRA_CMAKE_ARGS}
```

Comment:
I guess its good to be explicit about having a release build, but the default build type for TBB is release, isn't it?

Reply:
I saw `RelWithDebInfo` in my tests. There is no need to ship TBB's debug symbols out to all users of freud.

Reply:
Definitely!